RecoHGCal/TICL/plugins/TracksterP4FromEnergySumPlugin.cc

```diff
@@ -37,7 +37,7 @@ namespace ticl {
                                        edm::Event& event) const {
     // Find best vertex
     edm::Handle<std::vector<reco::Vertex>> vertex_h;
-    event.getByToken(vertex_token_, vertex_h);
+    vertex_h = event.getHandle(vertex_token_);
     auto vertex_coll = *vertex_h;
```

Comment:
Here it would be nice to be able to go directly to
```cpp
auto& vertex_coll = event.get(vertex_token_);
```
but that goes beyond the capabilities of the clang-tidy tool.

(btw, this line currently copies the `vector<Vertex>`)